src/EditorFeatures/VisualBasicTest/CodeActions/MoveType/MoveTypeTests.MoveToNewFile.vb

```diff
@@ -220,5 +220,44 @@ Partial Class Outer
 End Class"
             Await TestMoveTypeToNewFileAsync(code, codeAfterMove, expectedDocumentName, destinationDocumentText)
         End Function
+
+        <WorkItem(17879, "https://github.com/dotnet/roslyn/issues/17879")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsMoveType)>
+        Public Async Function DoNotMoveLeadingFileTrivia() As Task
+            Dim code =
+"
+''' <summary>
```

Comment:
Yep - just more common for VB with project level namespaces and imports.

Do we have tests that verify that this works?  I took a quick look and didn't see any.